.github/workflows/ci.yaml

```diff
@@ -54,6 +54,7 @@ jobs:
           - integration-scaling
           - integration-password-rotation
           - integration-tls
+          - integration-ha
```

Comment:
**todo:** To save on concurrency bugs that will inevitably come up, could we add a separate step here only for HA tests? e.g - https://github.com/canonical/zookeeper-operator/blob/test/ha_continuous/.github/workflows/ci.yaml

Reply:
Yes, I believe it is sane to keep them separated

Reply:
Actually, now that we start to have a lot of different tests (all inheriting the same env for the integration), it can make sense to simplify tox.ini file, as we do in data-platform libs, making it parametric, see https://github.com/canonical/data-integrator/blob/main/tox.ini#L64

Reply:
I present to you the funniest tests: `integration-ha-ha` :upside_down_face: Name will change once more are added, so it looks like `integration-ha-scaling` or something similar.